### DISCUSSION TITLE
Is it possible to have mapbox popup work like a normal div where I could select text or have an input field?

### DISCUSSION THREAD

**Discussion by Algirdyz** [visgl/deck.gl#5997] [Q&A]
I'm using mapbox popups with deck gl and right now clicking anywhere inside it except the close button creates a mapClicked event in Deck. Am I doing something incorrectly?

It also keeps the crosshair cursor over the popup.

```
<DeckGL
      ContextProvider={MapContext.Provider}
      controller={{ doubleClickZoom: false }}
      width={layoutState.dimensions.width || undefined}
      height={layoutState.dimensions.height || undefined}
      getCursor={() => 'crosshair'}
      layers={layers}
      ref={deck}
      onClick={(event) => {
           dispatch(mapEvent(event.coordinate));
      }}
  >
      <ReactMapGL
          ref={map}
      />
      <Popup longitude={4.4} latitude={51.8}>
          <div>Some text I want to select</div>
      </Popup>
</DeckGL>
```

**Comment by Pessimistress** [comment]
The popup by default inherits the container's `cursor` and `user-select` styles. You can either use `<Popup className="mypopup">` and add custom style in your stylesheet, or set inline styles on the popup's content `<div>`.

I cannot reproduce the click issue.

Example: https://codesandbox.io/s/recursing-stallman-tv7ef?file=/src/App.js
